contents/docs/self-host/deploy/other.mdx

```diff
@@ -37,9 +37,6 @@ ingress:
     enabled: true
 cert-manager:
   enabled: true
-kafka:
```

Comment:
This is the default since a while now. I think we simply forgot to remove it from here (it is not present for the other platforms)

Reply:
yes, thanks